### DISCUSSION TITLE
How do I set the page size measurements to millimeters

### DISCUSSION THREAD

**Discussion by danielsideas** [jgraph/drawio#4100] [Q&A]
Im using drawio / diagrams.net and are trying to map out a room where all the measurements for the room are in millimeters.

I started this by setting the units to mm so I could set the object sizes to the measurements however I tried to set the page size accordingly and notices that the only units available for  customisation are inches.  Is there a way to change this so I can update the page size as per my preferred unit

The diagram below shows where I tried to change the page sized.  

![Screenshot for page size customisation](https://github.com/jgraph/drawio/assets/2969576/6b9d8e30-4001-42c9-8139-891116aa697a)

Note I am aware that I can to a conversion from mm to inches but i prefer to have everything in MM

**Comment by AlexDaniel** [comment]
Now you can choose the unit. I don't know if it's truly useful, as all other measurements are still in pt, but it's better than nothing.
